src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import React from 'react';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const validTabId = tabs.some(tab => tab.id === activeTabId) ? activeTabId : tabs[0].id;
```

Comment:
```suggestion
  const validTabId = tabs.some(tab => tab.id === activeTabId) ? activeTabId : tabs[0].id;
```
looks redundant, we can just add the logic

```
const activeTab = tabs.find((tab) => tab.id === validTabId) || tabs[0]
```